app/models/provider_interface/provider_applications_page_state.rb

```diff
@@ -0,0 +1,33 @@
+module ProviderInterface
+  class ProviderApplicationsPageState
+    attr_reader :sort_order, :sort_by, :filter_visible, :filter_options
+
+    def initialize(params:)
+      @params = params
+      @sort_order = params[:sort_order].eql?('asc') ? 'asc' : 'desc'
+      @sort_by = params[:sort_by].presence || 'last-updated'
+      @filter_visible = params['filter_visible'] ||= 'true'
+      @filter_options = extract_filter_options
```

Comment:
`filter_options` only stores `status` values so the name may be a bit confusing - in future when we add a second filter it would need to be restructured or renamed anyway. Maybe just call it `status_filter_values`?